go.sum

```diff
@@ -3,6 +3,7 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
 github.com/fullstorydev/grpcurl v1.2.1 h1:5OUMseLFPJ7dXyvav6cB9z5HMr/vRdjeB4+LL9oGl4w=
 github.com/fullstorydev/grpcurl v1.2.1/go.mod h1:R6HR6EI02cafd3StLYD1Stwvqg3YaQCiXD8yj3Laq6g=
+github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b h1:VKtxabqXZkF25pY9ekfRL6a582T4P37/31XEstQ5p58=
```

Comment:
Do you mind reverting the changes to this file? They don't appear to be used/necessary.

Reply:
Fixed